src/api/Flooq.Api/Services/IUserService.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Flooq.Api.Services;
+
+public interface IUserService
+{
+  Task<ActionResult<IdentityUser>> GetUserById(string? id);
+  
+  Task<ActionResult<IdentityUser>> GetUserByEmail(string? email);
+  
```

Comment:
Like the controller: is it still needed? If yes, comments should be added.

Reply:
Yep, still needed